File: typescript-client-sdk/src/sdk/models/shared/security.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class SchemeApiKeyAuth extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "security, name=Authorization" })
  apiKey: string;
}


export class Security extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "security, scheme=true;type=apiKey;subtype=header" })
  apiKeyAuth: SchemeApiKeyAuth;
}


export class SchemeCodatLogin extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "security, name=Authorization" })
  authorization: string;
}

File: go-client-sdk/pkg/models/shared/security.go
package shared

type SchemeAPIKeyAuth struct {
	APIKey string `security:"name=Authorization"`
}

type Security struct {
	APIKeyAuth SchemeAPIKeyAuth `security:"scheme,type=apiKey,subtype=header"`
}

type SchemeCodatLogin struct {
	Authorization string `security:"name=Authorization"`
}
